File: mobile/src/main/java/com/windscribe/mobile/help/HelpPresenter.java
/*
 * Copyright (c) 2021 Windscribe Limited.
 */

package com.windscribe.mobile.help;

import android.content.Context;

public interface HelpPresenter {

    void init();

    void onDiscordClick();

    void onGarryClick();

    void onKnowledgeBaseClick();

    void onRedditClick();

    void onSendDebugClicked();

    void onSendTicketClick();

    void setTheme(Context context);
}

File: mobile/src/main/java/com/windscribe/mobile/help/HelpPresenter.kt
/*
 * Copyright (c) 2021 Windscribe Limited.
 */
package com.windscribe.mobile.help

import android.content.Context

interface HelpPresenter {
    fun init()
    fun onDiscordClick()
    fun onGarryClick()
    fun onKnowledgeBaseClick()
    fun onRedditClick()
    fun onSendDebugClicked()
    fun onSendTicketClick()
    fun setTheme(context: Context)
    suspend fun observeUserStatus()
}